src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -21,14 +21,15 @@ The following options can be configured on the root level of your browser-based
 The following can be configured as integration options in `new Replay({})`:
 
 | Key                     | Type                 | Default | Description                                                                                                                                                   |
-| ----------------------- | -------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ----------------------- | -------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------- |
 | stickySession           | boolean              | `true`  | Keep track of users across page loads. Note, because closing a tab ends the session, a single user using multiple tabs will be recorded as multiple sessions. |
 | mutationLimit           | number               | 10000   | The upper bound of mutations to process before Session Replay stops recording due to performance impacts. See [Mutation Limits](#mutation-limits)             |
 | mutationBreadcrumbLimit | number               | 750     | The upper bound of mutations to process before Session Replay sends a breadcrumb to warn of large mutations. See [Mutation Limits](#mutation-limits)          |
 | networkDetailAllowUrls  | (string \| RegExp)[] | `[]`    | Capture request and response details for XHR and fetch requests that match the given URLs.                                                                    |
 | networkCaptureBodies    | boolean              | `true`  | Decide whether to capture request and response bodies for URLs defined in `networkDetailAllowUrls`.                                                           |
 | networkRequestHeaders   | string[]             | `[]`    | Additional request headers to capture for URLs defined in `networkDetailAllowUrls`. See [Network Details](#network-details) to learn more.                    |
 | networkResponseHeaders  | string[]             | `[]`    | Additional response headers to capture for URLs defined in `networkDetailAllowUrls`. See [Network Details](#network-details) to learn more.                   |
+| beforeAddRecordingEvent | (event) => event     | null    | i => i                                                                                                                                                        | Filter additional recording events that include console logs and network requests/responses. |
```

Comment:
Looks like an extra column got in here. Will need the linter to run again for the whitespace.

```diff
- | beforeAddRecordingEvent | (event) => event     | null    | i => i                                                                                                                                                        | Filter additional recording events that include console logs and network requests/responses. |
+ | beforeAddRecordingEvent | (event) => event     | null    | Filter additional recording events that include console logs and network requests/responses. |
```